Description: Rename minitest keyword value used in test
 Test uses the keyword value as a variable. Replace it with value1
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-07-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/redis/store_test.rb
+++ b/test/redis/store_test.rb
@@ -23,13 +23,13 @@
   describe '#set' do
     describe 'with expiry' do
       let(:key) { 'key' }
-      let(:value) { 'value' }
+      let(:value1) { 'value' }
       let(:options) { { :expire_after => 3600 } }
 
       it 'must not double marshall' do
         Marshal.expects(:dump).once
 
-        @store.set(key, value, options)
+        @store.set(key, value1, options)
       end
     end
   end
@@ -37,13 +37,13 @@
   describe '#setnx' do
     describe 'with expiry' do
       let(:key) { 'key' }
-      let(:value) { 'value' }
+      let(:value1) { 'value' }
       let(:options) { { :expire_after => 3600 } }
 
       it 'must not double marshall' do
         Marshal.expects(:dump).once
 
-        @store.setnx(key, value, options)
+        @store.setnx(key, value1, options)
       end
     end
   end
